src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1581,6 +1618,119 @@ public void TestConnectionStringEnvVarHasHighestPrecedence()
             }
         }
 
+        /// <summary>
+        /// Validates that deserialization of config file is successful for the following scenarios:
+        /// 1. Nested Mutations section is null
+        /// {
+        ///     "nested-mutations": null
+        /// }
+        /// 
+        /// 2. Nested Mutations section is empty.
+        /// {
+        ///     "nested-mutations": {}
+        /// }
+        ///
+        /// 3. Create field within Nested Mutation section is null.
+        /// {
+        ///     "nested-mutations": {
+        ///         "create": null
+        ///     }
+        /// }
+        ///
+        /// 4. Create field within Nested Mutation section is empty.
+        /// {
+        ///     "nested-mutations": {
+        ///         "create": {}
+        ///     }
+        /// }
+        /// 
+        /// For all the above mentioned scenarios, the expected value for NestedMutationOptions field is null.
+        /// </summary>
+        /// <param name="baseConfig">Base Config Json string.</param>
+        [DataTestMethod]
+        [DataRow(TestHelper.BASE_CONFIG_NULL_NESTED_MUTATIONS_FIELD, DisplayName = "Validate successful deserialization when nested mutation section is null")]
```

Comment:
more self descriptive displayname:
Instead of: "Validate successful deserialization",
i propose:
"NestedMutationOptions field deserialized as null when {test case}"